deploy/helm/node-collector-config.yaml

```diff
@@ -294,13 +294,21 @@ processors:
       - context: metric
         statements:
           - extract_sum_metric(true) where (name == "k8s.istio_request_bytes" or name == "k8s.istio_response_bytes" or name == "k8s.istio_request_duration_milliseconds")
+          - extract_count_metric(true) where (name == "k8s.istio_request_duration_milliseconds")
           - set(name, "k8s.istio_request_bytes.rate") where name == "k8s.istio_request_bytes_sum"
           - set(name, "k8s.istio_response_bytes.rate") where name == "k8s.istio_response_bytes_sum"
-          - set(name, "k8s.istio_request_duration_milliseconds.rate") where name == "k8s.istio_request_duration_milliseconds_sum"
           - set(name, "k8s.istio_requests.rate") where name == "k8s.istio_requests_total"
           - set(name, "k8s.istio_tcp_sent_bytes.rate") where name == "k8s.istio_tcp_sent_bytes_total"
           - set(name, "k8s.istio_tcp_received_bytes.rate") where name == "k8s.istio_tcp_received_bytes_total"
   
+  experimental_metricsgeneration/istio-metrics:
+    rules:
+      - name: k8s.istio_request_duration_milliseconds.rate
```

Comment:
This is not a rate. This is an average over the entire history of collection of this metric. I'm not sure that's what we want.

Reply:
it is not rate at this row, but it should be rate after other processors are called